hecuba_py/src/hdict.py

```diff
@@ -493,6 +494,8 @@ def make_persistent(self, name):
                     log.error("Error creating the Qbeast trigger: %s %s", trigger_query, ex)
                     raise ex
 
+        self._store_meta(self._build_args)
```

Comment:
Please leave the `store_meta` at the end of the method. In this way we make sure that another process does not believe the object is persisted before we write the data.